etc/config/c++.amazon.properties

```diff
@@ -1185,7 +1185,7 @@ compiler.hexagonclang1605.compilerCategories=clang-hexagon
 
 ################################
 # Clang for clad
-group.clad-clang.compilers=clang1810_clad180:clang1810_clad_trunk
+group.clad-clang.compilers=clang1810_clad180:clang1910_clad190:clang1910_clad_trunk
```

Comment:
By removing the `clang1810_clad_trunk` this invalidates any URLs folks will have saved that pointed at that compiler.  Is there any benefit to keeping both 18.1.0 trunk _and_ 19.1.0 trunk? Or should we auto-update any link that points to `clang1810_clad_trunk` to `clang1910_clad_trunk` ? Will there ever be a time when the clad trunk should have both versions of clang available?

Reply:
I don't think we would want 2 versions of a trunk version. If it is possible to migrate URLs to point to the newer version of clad, that would be good.
Pinging @vgvassilev

Reply:
Fantastic; if that's the case can I suggest we:
- rename `clang1910_clad_trunk` to just be `clang_latest_clad_trunk` (or similar)
- add `compiler.clang_latest_clad_trunk.alias=clang1810_clad_trunk` to redirect any old URLs/references to the new "latest"

If we ever think we'll need to have more than one trunk, then keep the name as-is but add the `compiler.clang1910_clad_trunk.alias=clang1810_clad_trunk` here to migrate the URLs.

Reply:
Another option would be to rename the truck version to be independent of the clang version, something like `clad_trunk`. Then URLs will remain valid throughout.
And the associated clang version will be the latest/highest version we support.

Reply:
Yes: that's what I meant by "clang_latest_clad_trunk" but also just "clad_trunk" works too.

Reply:
The changes you suggested are brilliant. It is late night at my place. Will update the PR tomorrow morning. Hope that's fine.

Reply:
Of course! No rush on my side; if you are OK I can make the relevant changes, but also we can wait another day :) I'll be around to merge (I have been away for a few days hence quiet)